src/comments.js

```diff
@@ -46,7 +46,14 @@ export default class Mastodon extends HTMLElement {
   async fetchComments(url) {
     const tootUrl = new URL(url);
     const { origin, pathname } = tootUrl;
-    const [_, id] = pathname.match(/\/(\d+)$/);
+
+    let id = pathname.match(/\/(\d+)$/);
+
+    // In case it’s a a Pleroma server, with /notice/ URLs.
+    if (pathname.includes("/notice/")) {
+      id = pathname.replace('/notice/', '');
```

Comment:
I'd do something more consistent here. This will fail with `/notice/1234598?name=value`. Maybe this regular expression:

```js
[, id] = pathname.match(/^\/notice\/([^\/?#]+)/);
```